hawk/src/test/java/com/orhanobut/hawk/HawkTest.java

```diff
@@ -44,7 +46,7 @@ public HawkTest() {
     context = Robolectric.buildActivity(Activity.class).create().get();
   }
 
-  @Override public void setUp() throws Exception {
+  @Override @Before public void setUp(){
```

Comment:
maybe I missed something but on my machine Before and After were not working. that method did not run for me